node/core/candidate-validation/src/lib.rs

```diff
@@ -634,7 +633,7 @@ trait ValidationBackend {
 			// Wait a brief delay before retrying.
 			futures_timer::Delay::new(PVF_EXECUTION_RETRY_DELAY).await;
 
-			gum::debug!(
+			gum::info!(
```

Comment:
What should the operator do with this information? Do we log the worker death error somewhere?

Info is also probably not seen, you would need to make this `warn!` to appear in monitoring.

Reply:
I think there is not enough information attached to make anything out of this. Even if they open an issue for this, we will be clueless.

Reply:
There wasn't any more information available here, but we can log the error when it happens!

(I raised a commit adding a log, but I just realized that I did it for preparation only, which won't catch AmbiguousWorkerDeath - gimme a sec.)

> Info is also probably not seen, you would need to make this warn! to appear in monitoring.

Done!